frontend/src/metabase/parameters/utils/parameter-values.js

```diff
@@ -1,11 +1,7 @@
 import { getParameterType } from "metabase-lib/parameters/utils/parameter-type";
 import { hasParameterValue } from "metabase-lib/parameters/utils/parameter-values";
 
-export function getParameterValueFromQueryParams(
-  parameter,
-  queryParams,
-  metadata,
-) {
+export function getParameterValueFromQueryParams(parameter, queryParams) {
```

Comment:
`getParameterValueFromQueryParams` also didn't use its `metadata` arg, so I removed it.